easybuild/easyconfigs/x/XDS/XDS-3Nov2014-INTEL64_Linux_x86_64.eb

```diff
@@ -0,0 +1,39 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Adam Mazur
+# Research IT
+# Biozentrum - University of Basel
+
+easyblock = "Tarball"
+
+name = 'XDS'
+version = '3Nov2014'
+versionsuffix = '-INTEL64_Linux_x86_64'
+
+homepage = 'http://xds.mpimf-heidelberg.mpg.de/'
+description = """ X-ray Detector Software for processing single-crystal
+ monochromatic diffraction data recorded by the rotation method. For a
+ description of the package, including references, visit xds_doc. """
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+# source_urls = ['ftp://ftp.mpimf-heidelberg.mpg.de/pub/kabsch']
+# Download does not provide version information. Downloaded manually
+# and renamed with the date as version (check the main website for current
+# version number e.g. "3Nov2014")
+
+sources = ['%(name)s-%(version)s%(versionsuffix)s.tar.gz']
+
+sanity_check_paths = {
+    'files': ["xds", "xdsconv"],
+    'dirs': []
+}
+
+modextrapaths = {
+    'PATH': ''
+}
+
+modextravars = {
+    'KMP_STACKSIZE': '8m'
```

Comment:
```
The first line makes the executables of the XDS package accessible, while the second line defines the stack
size limit of the Linux POSIX threads libraries used by OpenMP for xds_par. (The limit 8m may need to be
increased if xds_par collapses with a segmentation fault in the COLSPOT or INTEGRATE step for some of
your data sets.)
```

At least add this as a comment in the easyconfig + a reference where you found it?